### PR TITLE
Changing namespaces

### PR DESCRIPTION
**Pull request by WojcikMike**
Connect to Particular/PlatformDevelopment#719

### CODE REVIEW

**Comment by WojcikMike** [comment]
@Particular/rabbitmq-transport-maintainers please review and merge. At this time we have 3 namespaces:
- NServiceBus
 - RabbitMQTransport
 - RabbitMQTransportSettingsExtensions
- NServiceBus.Transports.RabbitMQ.Routing
 - IRoutingTopology
- NServiceBus.Transports.RabbitMQ
 Other

**Comment by bording** [comment]
Looking at our [component naming](https://github.com/Particular/PlatformDevelopment/blob/master/designprinciples/nservicebus/component-naming.md) and [namespace](https://github.com/Particular/PlatformDevelopment/blob/master/designprinciples/nservicebus/namespaces.md) docs we should have two namespaces:

- NServiceBus
- NServiceBus.Transport.RabbitMQ

**Comment by WojcikMike** [comment]
@bording changes made, now we have only 2 namespaces as you specified.

**Comment by bording** [comment]
I pushed up a tweak to the new obsolete attribute and also went ahead and adjusted the namespaces of the AcceptanceTest project.

Merging!

**Comment by SimonCropp** [comment]
Is there a related doco pr? Should any of this be in the upgrade guide?

**Comment by bording** [comment]
> Is there a related doco pr? Should any of this be in the upgrade guide?

Since we're working on getting beta1 ready, updating samples, snippets, etc is part of the PoA for https://github.com/Particular/PlatformDevelopment/issues/719

File: src/NServiceBus.RabbitMQ.Tests/APIApprovals.Approve.approved.txt [hunk 1]
```diff
@@ -11,25 +11,23 @@ namespace NServiceBus
...
-        [System.ObsoleteAttribute("Replaced by NServiceBus.Callbacks package. Will be removed in version 5.0.0.", true)]
+        [System.ObsoleteAttribute("Replaced by NServiceBus.Callbacks package. The member currently throws a NotImple" +
+            "mentedException. Will be removed in version 5.0.0.", true)]
         public static NServiceBus.TransportExtensions<NServiceBus.RabbitMQTransport> DisableCallbackReceiver(this NServiceBus.TransportExtensions<NServiceBus.RabbitMQTransport> transportExtensions) { }
         public static NServiceBus.TransportExtensions<NServiceBus.RabbitMQTransport> TimeToWaitBeforeTriggeringCircuitBreaker(this NServiceBus.TransportExtensions<NServiceBus.RabbitMQTransport> transportExtensions, System.TimeSpan waitTime) { }
-        [System.ObsoleteAttribute("Will be removed in version 5.0.0.", true)]
+        [System.ObsoleteAttribute("The member currently throws a NotImplementedException. Will be removed in version" +
```
Comment (adamralph) [inline src/NServiceBus.RabbitMQ.Tests/APIApprovals.Approve.approved.txt]: @WojcikMike out of interest, why did this message change in this changeset?

Comment (SimonCropp) [inline src/NServiceBus.RabbitMQ.Tests/APIApprovals.Approve.approved.txt]: @adamralph update to the newest obsoleteEx

Comment (adamralph) [inline src/NServiceBus.RabbitMQ.Tests/APIApprovals.Approve.approved.txt]: @SimonCropp that would make sense, but I don't see it anywhere in this changeset.

@Particular/rabbitmq-transport-maintainers does this mean the build was failing before this PR was merged?

Comment (bording) [inline src/NServiceBus.RabbitMQ.Tests/APIApprovals.Approve.approved.txt]: @adamralph I updated to the latest ObsoleteEx, but I forgot about the api approver being added to the tests project, so I didn't update it.

The tests weren't being run as part of a build, so it wasn't caught. @WojcikMike updated TC to include those in the build, and that's why they are now showing up here.

Comment (adamralph) [inline src/NServiceBus.RabbitMQ.Tests/APIApprovals.Approve.approved.txt]: @bording :+1: all makes sense now, thanks.

**Pull request merged**